app/src/main/res/values-tr/params_info.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- VM -->
+    <string name="admin_reserve_kbytes">Cap_sys_admin özelliğine sahip kullanıcılar için ayrılması gereken sistemdeki boş bellek miktarı</string>
+    <string name="block_dump">Sıfır olmayan bir değere ayarlandığında blok G/Ç hata ayıklamasını etkinleştirir</string>
+    <string name="compact_memory">Yalnızca CONFIG_COMPACTION ayarlandığında kullanılabilir. Dosyaya 1 yazıldığında, tüm bölgeler, mümkün olduğunda bitişik bloklarda boş bellek mevcut olacak şekilde sıkıştırılır.</string>
+    <string name="compact_unevictable_allowed">Yalnızca CONFIG_COMPACTION ayarlandığında kullanılabilir. 1 olarak ayarlandığında, sıkıştırmanın sayfaların sıkıştırılması için kaçınılmaz lru'yu (kilitli sayfalar) incelemesine izin verilir. Sıkıştırmanın önlenemeyen sayfaları hareket ettirmesini önlemek için 0 olarak ayarlayın. Varsayılan değer 1'dir.</string>
```

Comment:
This string resource contains an unescaped apostrophe (`'`).
Same for lines 10, 13, 15, 16, 20, 21, 22, 27, 29, 30 and 32.

Suggestion: fix apostrophe escaping by using a `\` character before each.